package.json

```diff
@@ -156,6 +254,6 @@
     "metals-languageclient": "^0.1.9",
     "promisify-child-process": "^3.1.3",
     "semver": "^7.1.1",
-    "vscode-languageserver-protocol": "^3.15.1"
+    "vscode-languageserver-protocol": "3.15.0-next.6"
```

Comment:
Do you have compilation error in file index.ts in lines with `client.onRequest`, `client.sendNotification`? This change fixes error. I took this version from coc.nvim.

Reply:
hmmm, no, I haven't had any issues with it.

Reply:
I believe this is why you declared `client` as `any`:
https://github.com/scalameta/coc-metals/blob/master/src/index.ts#L218-L224